CollectionsInterface/Exercise5/src/Main.java

```diff
@@ -1,6 +1,38 @@
+import java.util.ArrayList;
+import java.util.List;
+
 public class Main {
 
+    public static void deleteRepeatedCars(ArrayList<Car>cars){
+        ArrayList<Car>aux = new ArrayList<>();
+        while(!cars.isEmpty()){
+            Car value = cars.get(cars.size()-1);
+            while(cars.contains(value)){
+                value = cars.remove(cars.indexOf(value));
+                if(!aux.contains(value)){
+                    aux.add(value);
+                }
+            }
+        }
+        cars.addAll(aux);
+    }
```

Comment:
Nunca se me hubiese ocurrido hacerlo de esa manera. Lo unico que me hace un poco de ruido es que primero borras todas las ocurrencias de ese `Car` y despues agregas una. Yo hubiese optado por retornar la lista sin los repetidos. Te dejo un ejemplo
```suggestion
    public static List<Car> deleteRepeatedCars(List<Car>cars){
        List<Car> aux = new ArrayList<>();
        for (Car car: cars) {
            if(!aux.contains(car)){
                aux.add(car);
            }
        } 
       return aux;
    }
```

Reply:
Estuve más de lo esperado pensándolo jajaja, había pensado en retornarlo al arreglo sin los repetidos pero no estaba seguro si estaba permitido por la consigna. Igual lo super simplificaste, no me di cuenta de hacerlo así. Gracias!